usecases.md

```diff
@@ -25,15 +23,11 @@ Framework functional capabilities are grouped into three additional categories:
 1. [Administer the framework](#administer)
 1. [Archive data and forecasts](#archive)
 
-Metrics, benchmark forecasts, data formats, data sharing policies, and legal considerations will be detailed in other documents.
-
+**For brevity, a *forecast* refers to the *[forecast evaluation time series](/definitions/#forecastevalts)* defined in the [Definitions section](/definitions/).**
 
-## Use cases  {#usecases}
```

Comment:
I think we need to keep this heading but put it right above `### 1. Evaluate forecasts`. Otherwise the numbered use cases (level 3 headings) are technically under the "Purpose and Summary" (level 2 heading).

Reply:
My error